docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1990,7 +1990,8 @@ Census
 
 .. note::
 
-  In the final version of the observers, following the noise functions, please have age as an integer and all other data as strings. 
+  In the final version of the observers, following the noise functions, please have all data as strings. Age must be rounded to the nearest whole number.
```

Comment:
```suggestion
  In the final version of the observers, following the noise functions, please have all data as strings. Age must be rounded down to a whole number before applying noise.
```

Alternatively, it might be clearer to add this note directly to the table row for age.

Reply:
This clarification helps, thanks!